pkg/sink/initialization.go

```diff
@@ -133,3 +135,26 @@ func ConfigureClients(clusterConfig *rest.Config) (Clients, error) {
 		TriggersClient:  triggersClient,
 	}, nil
 }
+
+var elBackoff = wait.Backoff{
+	Duration: 50 * time.Millisecond,
+	Factor:   2.0,
+	Jitter:   0.3,
+	Steps:    10,
+	Cap:      5 * time.Second,
+}
+
+// WaitForEventListenerOrDie waits for the Kubernetes eventlistener configuration
+func (r *Sink) WaitForEventListenerOrDie() {
```

Comment:
Can this elBackOff be a param that is passed to this function? That will also make the test a bit more easier to follow since we won't be modifying the variable in this package to test.